### PR TITLE
Option to Disable the Checkbox

### PR DESCRIPTION
**Pull request by KapuzenSohn**
I needed a Option to Disable a Checkbox. Thats why I added one

### CODE REVIEW

**Comment by KapuzenSohn** [comment]
Issue Number is #30

**Comment by jspdown** [comment]
Thanks @KapuzenSohn. I agree with you, this option should implemented.

File: src/checkbox/checkbox.js [hunk 1]
```diff
@@ -54,20 +54,30 @@ angular.module('angularify.semantic.checkbox', [])
                 scope.checked = true;
                 element.children()[0].setAttribute('checked', '');
             }
+            
+            //
+            // check if the parameter disabled is available
+            //
+            if (scope.disabled == 'disabled') {
```
Comment (jspdown) [inline src/checkbox/checkbox.js]: I think you forgot add `disabled` to the isolated scope. It should be declared in line 10-14 of this file.

**Comment by KapuzenSohn** [comment]
Oh sorry I forgot.
I will add it when I am home.

**Comment by KapuzenSohn** [comment]
Closed for #36

**Pull request closed without merging**